include/service/entities/plugin_service.js

```diff
@@ -1277,28 +1316,53 @@ module.exports = function PluginServiceModule(pb) {
         var hasDependencies = true;
         var tasks = util.getTasks(Object.keys(plugin.dependencies), function(keys, i) {
             return function(callback) {
-                
+
                 //verify that the module exists and its package definition is available
                 var packagePath = path.join(PluginService.getPluginsDir(), plugin.dirName, 'node_modules', keys[i], 'package.json');
-                var modulePath = packagePath;
-                fs.exists(modulePath, function(exists) {
-                    if (!exists) {
-                        pb.log.warn('PluginService: Plugin %s is missing dependency %s', plugin.name, keys[i]);
-                        
-                        hasDependencies = false;
-                        return callback(null, false);
+                var rootPackagePath = path.join(process.cwd(), 'node_modules', keys[i], 'package.json');
```

Comment:
Is the assumption that the node_modules directory will always be at the directory where PB is installed?  If so then it might be better to force that assumption: ```path.join(pb.config.docRoot, 'node_modules', keys[i], 'packag.json');```

Either way this implementation is fine as is.